odoo/orm/fields.py

```diff
@@ -234,6 +234,7 @@ def _read_group_many2one_field(self, records, domain, order):
     type: str                           # type of the field (string)
     relational = False                  # whether the field is a relational one
     translate = False                   # whether the field is translated
+    falsy_value = None                  # falsy value for comparisons (optional)
```

Comment:
This attribute name is too generic, while `falsy_value` is very specific to search.
Couldn't we replace this attribute by some method that takes a collection of values and returns the non-falsy values with a flag indicating the presence of falsy values?
```py
if operator == 'in':
    values, is_null = self._prepare_in_values(value)  # here
    sql = SQL("FALSE")
    if values:
        sql = SQL("%s IN %s", ..., tuple(_to_column(v) for v in values))
    if is_null:
        sql = SQL("(%s OR %s IS NULL)", sql, ...)
```